### PR TITLE
add CRS.geodetic_crs as replacement for CRS.to_geodetic()

### PR DESCRIPTION
**Pull request by snowman2**
#297

### CODE REVIEW

**Comment by snowman2** [comment]
Also, made the private attributes private.

File: pyproj/_crs.pyx [hunk 1]
```diff
@@ -1316,20 +1339,14 @@ cdef class _CRS(Base):
...
         """
-        cdef PJ * projobj
-        projobj = proj_crs_get_geodetic_crs(self.projctx, self.projobj)
-        if projobj == NULL:
-            return None
-        try:
-            return self.__class__(_to_wkt(self.projctx, projobj))
-        finally:
-            proj_destroy(projobj) # deallocate temp proj
+        return self.geodetic_crs
```
Comment (jorisvandenbossche) [inline pyproj/_crs.pyx]: Add deprecation warning?

Comment (snowman2) [reply pyproj/_crs.pyx]: Probably better to deprecated it. I had the thought as well.

Comment (snowman2) [reply pyproj/_crs.pyx]: Just added deprecation warning.

**Pull request merged**